src/main/java/com/karimbouchareb/chromatographysimulator/ChromatographySimulatorApp.java

```diff
@@ -124,7 +125,7 @@ public static void main(String[] args) {
      * screen with each of its ui components.
      */
     @Override
-    public void init() {
```

Comment:
Did you already push a commit that creates ChromatographySimulatorApp.java ImageView's makeImageView(String filePath) method?

Reply:
Im not sure I understand what you're asking

Reply:
Sorry - in ChromatographySimulatorApp.java there's a method called `makeImageView(String filePath)`. I don't think that method existed before this PR but I don't see where it was added. Do you know when this was added?

Reply:
Oh - maybe it did already exist before this PR. I don't recall.

Reply:
Oh yeah it was written long long ago

Reply:
Whoops ok, ignore previous comments.